ui/src/components/shared/CredentialRevokeModal.tsx

```diff
@@ -29,18 +31,7 @@ export function CredentialRevokeModal({
         onClose();
         onRevoke();
```

Comment:
This is missing (here or inside the callback itself) the additional notification that the credential has been revoked. See [comment](https://www.figma.com/file/BE3tUbvtSB2EqmJsybpeFC?node-id=5808:184164#417295665).

Reply:
Sorry, but I don't get what it's missing, you are pointing to a line without changes and the notification is below that line

Reply:
We need a `message.success` that displays "Credential successfully revoked" (or whatever the BE returns after revocation) in addition to the issuer state notification.

Reply:
I think that is more related to the decision that you'll  reach in figma, if we change it to an info message it'll has sense but if not I think that we can leave with just this one

Reply:
Just for consistency though, we should always return the operation's message regardless of the issuer state notification.